src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -504,6 +504,10 @@ protected bool SkippedType(INamedTypeSymbol type)
 					// Skipped to include generic variance.
 					return true;
 
+				case "Windows.Foundation.Collections.ValueSet":
```

Comment:
This is actually a problem with sync generator, it should ignore the nullables, right @jeromelaban ? Maybe worth creating an issue to remove this after it is merged and to fix it in generator.

Reply:
Seems like this comment is not directly related to this PR, so I think I can "resolve conversation".